spec/features/admin_stats_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.feature "Admin stats" do
+  before do
+    @submitted_claims = create_list(:claim, 6, :submitted)
+    @approved_claims = create_list(:claim, 4, :approved, submitted_at: 10.weeks.ago)
+    @unfinished_claims = create_list(:claim, 1, :submittable)
+    @claims_approaching_deadline = create_list(:claim, 2, :submitted, submitted_at: 5.weeks.ago)
+    @claims_passed_deadline = create_list(:claim, 1, :submitted, submitted_at: 10.weeks.ago)
+    sign_in_to_admin_with_role(AdminSession::SERVICE_OPERATOR_DFE_SIGN_IN_ROLE_CODE)
+    visit admin_root_path
+  end
+
+  scenario "Service operator is shown how many claims have been submitted" do
```

Comment:
Can we combine these four scenarios into a single "Service operator is shown claim stats"? Each scenario has exactly the same expensive setup and is checking the same page. We only need to create the records, log the user in and render the page once to verify the expected behaviour. That'll make reduce the hit on our test suite run time a chunk.